tests/ember-app/node-tests/build.test.mjs

```diff
@@ -54,11 +58,48 @@ async function hasJSSourceMapsURL(expectation) {
   return await hasSourceMappingURL('dist/**/*.js', expectation);
 }
 
+async function appJS() {
+  // based of package.json#name
+  let files = await globby('**/ember-app-*.js', { expandDirectories: true });
+
+  if (files.length === 0) {
+    throw new Error('No app js file found');
+  }
+
+  // there can only be one app js file
+  let data = await fs.readFile(files[0]);
+  return data.toString();
+}
+
+async function vendorJS() {
+  let files = await globby('**/vendor-*.js', { expandDirectories: true });
+
+  if (files.length === 0) {
+    throw new Error('No vendor js file found');
+  }
+
+  // there can only be one app js file
+  let data = await fs.readFile(files[0]);
+  return data.toString();
+}
+
 test('basic build', async () => {
   await build();
 
   expect(exists('dist/**/*.map')).toBeFalsy();
   expect(await hasJSSourceMapsURL(false)).toBeFalsy();
+
+  // assert that the image references end with a fingerprint
+  // (because we don't want to interfece with broccoli-asset-rev)
+  let app = await appJS();
+  let vendor = await vendorJS();
+
+  // fingerprint occurred
```

Comment:
these are the new assertions -- and I also verified this manually via ember s and opening the inspector